AnnotationTool/myapp/static/js/upload_image.js

```diff
@@ -0,0 +1,19 @@
+      function previewImage(event) {
+        var input = event.target;
+        var preview = document.getElementById('preview');
+        var previewContainer = document.getElementById('imagePreview');
+
+        if (input.files && input.files[0]) {
```

Comment:
Is it really necessary to check if a file was selected like that or is there another way?